pkg/store/store.go

```diff
@@ -31,11 +31,13 @@ var (
 	buildDate                = ""
 	gitCommit                = ""
 	SegmentWriteKey          = ""
-	maxDefVersion            = "2.1.1"
-	RuntimeDefURL            = "manifests/runtime.yaml"
+	maxDefVersion            = "2.1.2"
```

Comment:
please do not touch this field it is deprecated, it's only here to allow to install runtimes with version < xyz